collectors/okta/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "okta-collector",
-  "version": "1.1.5",
+  "version": "1.1.6",
```

Comment:
you also need to bump PAWS dependency version

Reply:
I guess we'll need to do this after we release the paws framework, otherwise the tests wont pass.

Reply:
so let's have a separate PR for `paws-collector` change.